types/office-js-preview/index.d.ts

```diff
@@ -17746,11 +17746,17 @@ declare namespace Office {
          *
          * **Errors**:
          *
-         * - `HTTPRequestFailure`: The request has failed. Please look at the diagnostics object for the HTTP error code.
+         * - `GenericTokenError: An internal error has occurred.` - In Exchange Online environments, this error occurs when the token can't be retrieved because legacy Exchange tokens
+         * for Outlook add-ins are turned off. We recommend using NAA as a single sign-on solution for your add-in. For guidance on how to implement NAA, see
+         * {@link https://learn.microsoft.com/office/dev/add-ins/develop/enable-nested-app-authentication-in-your-add-in | Enable SSO in an Office Add-in using nested app authentication}.
```

Comment:
Again I think we should link to the FAQ https://aka.ms/naafaq as that will have more comprehensive information about deprecation of the tokens, or how to turn them on/off if someone wants to do that.

Reply:
Updated.